db/api.sql

```diff
@@ -84,6 +84,7 @@ create table suppression_files (
 create table suppressions (
     id serial primary key,
     file_id integer not null,
+    blue_button_id text,
```

Comment:
Maybe `varchar(15)`? https://bluebutton.cms.gov/resources/variables/bene_id/

Reply:
We could do this but Im unsure of any unintended consequences of changing this field type..(thinking db migration stuff). The same thing also applies to our cclf_beneficiaries table too, (where it was copied from). And also unsure if this change should happen in the gorm model itself?? 2 sources of truth..ahhhh

Reply:
@embh Thoughts?